crates/nu-command/tests/commands/random/chars.rs

```diff
@@ -3,7 +3,7 @@ use nu_test_support::nu;
 #[test]
 fn generates_chars_of_specified_length() {
     let actual = nu!(r#"
-        random chars --length 15 | str stats | get chars
+        random chars --length 15 | str stats | get chars.0
```

Comment:
I don't see the point of it chunking any `String`/`ByteStream`. `str stats` is inherently a reduction, we should `Ctrl-C` correctly, but don't think that makes the usability any better of having to pull the result from potentially multiple records in a list(stream).

Reply:
I think the point is, if you're trying to run `str stats` on a file, don't read the entire file into memory. I think that was the complaint when running on multi-gigabyte files.